scrapy_zyte_api/handler.py

```diff
@@ -22,6 +24,24 @@
 logger = logging.getLogger(__name__)
 
 
+def _truncate(obj, limit):
+    if isinstance(obj, dict):
+        for key in list(obj):
+            value = obj[key]
+            if isinstance(value, str):
+                if len(value) > limit:
+                    obj[key] = value[: limit - 1] + "…"
+            elif isinstance(value, (list, dict)):
```

Comment:
Do we also allow and process any other API request parameters that are iterables like `tuples` and `sets`?

Reply:
Since it is a JSON API, and we use `json` from the standard library to parse it, I think it is safe to assume we will only have to handle lists and dicts here.